src/inverseRobot.js

```diff
@@ -24,7 +24,25 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const robot2 = {};
+  const keys = Object.keys(robot).reverse();
+  const vals = Object.values(robot);
+
+  for (let i = 0; i < vals.length; i++) {
+    for (let j = i + 1; j < vals.length; j++) {
+      if (vals[i] === vals[j]) {
+        return null;
+      }
+    }
```

Comment:
instead of this, you can just check if object has a property which you want to add in loop below